src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallGlobalOrToolPathCommand.cs

```diff
@@ -157,6 +158,17 @@ private int ExecuteInstallCommand(PackageId packageId)
 
             IToolPackage oldPackageNullable = GetOldPackage(toolPackageStoreQuery, packageId);
 
+            if (oldPackageNullable != null)
+            {
+                NuGetVersion nugetVersion = GetBestMatchNugetVersion(packageId, versionRange, toolPackageDownloader);
+
+                if (ToolVersionAlreadyInstalled(oldPackageNullable, nugetVersion))
+                {
+                    _reporter.WriteLine(string.Format(LocalizableStrings.ToolAlreadyInstalled, _packageId).Green());
```

Comment:
Small nit: it might be good to supplement this message to say:
Tool '{0}' is already installed with version '{1}'.

Reply:
Agree and done!